cogboard-webapp/src/components/Widget/index.js

```diff
@@ -150,6 +155,14 @@ const Widget = ({ id, index }) => {
           widgetTypeData={widgetTypeData}
         />
       </AppDialog>
+      <ConfirmationDialog
+          open={confirmationDialogOpened}
+          title={`Delete ${title}`}
+          content={`Are you sure you want to delete ${title}?`}
+          handleOk={deleteWidget}
+          labelOk={`Delete`}
```

Comment:
Backticks notation here is not necessary, it can be just `labelOk="Delete"`, but still ok to me

Reply:
Fixed